src/main/java/com/werfen/report/service/template/BaseReportTemplateBuilder.java

```diff
@@ -50,7 +49,7 @@
     private static final int HEADER_FRAME_Y = 0;
 
     private static final int FOOTER_TEXT_FIELD_HEIGHT = 9;
-    private static final int HEADER_FIELD_1_X = HEADER_MARGIN_X;
+    private static final int HEADER_FIELDS_INITIAL_X = HEADER_MARGIN_X;
```

Comment:
Sorry for being picky, but I would move this header constant together with the group of header constants above. Same for later footer constant.

Reply:
Done!! Thank you for the review.